### PR TITLE
Update script.js

### PR DESCRIPTION
**Pull request by joelp999**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment? 1.5

Please fill in one error and/or error message you received while working on this assignment. Unexpected Token "("

What part of the assignment did you spend the most time on? Fixing errors with the number calculation

Comfort Level (1-5): 4

Completeness Level (1-5): 4

What did you think of this deliverable? I think the output values could be condensed a bit better because it's very repetitive.

Is there anything in this code that you feel pleased about? It works

### CODE REVIEW

**Comment by joelp999** [comment]
I added the following functions to the script: username input, tally of games lost/won.

File: script.js [hunk 1]
```diff
@@ -1,4 +1,135 @@
...
+  if (randomNumber == 1) {
+    return PAPER;
+  }
+  if (randomNumber == 2) {
+    return SCISSORS;
+  }
+  if ((randomNumber = 3)) {
+    return STONE;
+  }
+};
```
Comment (Ennnm) [inline script.js:23]: Nice condensing this into one function

File: script.js [hunk 2]
```diff
@@ -1,4 +1,135 @@
...
+    totalCount = totalCount + 1;
+    drawCount = drawCount + 1;
+    return (
+      "The computer chose scissors.<br>You chose scissors.<br>You draw. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
```
Comment (Ennnm) [inline script.js:133]: For now each win/loss condition is enumerated and the output values produced one by one, why is quite a lot of code duplication
Can consider grouping all the win condtions under one if condition using ||
e.g. 
if (
(input == SCISSORS && computerChoiceMade == SCISSORS) ||
(input == SCISSORS && computerChoiceMade == STONE)  ||
(input == SCISSORS && computerChoiceMade == PAPER))

and puting the inputs and computerChoiceMade as template literals in your returned string

`The computer chose ${computerChoiceMade}........You chose ${input}`

**Review by Ennnm — COMMENTED** [review]
Good effort 🎈

**Pull request closed without merging**